examples/src/main/java/io/snappydata/examples/JavaAirlineDataJob.java

```diff
@@ -28,7 +28,7 @@
   public static void main(String[] args) throws Exception {
     SparkConf sparkConf = new SparkConf().setAppName("JavaSparkSQL").setMaster("local[2]");
     JavaSparkContext jsc = new JavaSparkContext(sparkConf);
-    SnappyContext snc = SnappyContext.getOrCreate(jsc);
+    SnappyContext snc = new SnappyContext(jsc.sc());
```

Comment:
"new" seems unnecessary. Perhaps can make it protected so everywhere apply is used in uniform way.